Libraries/LibVT/TerminalWidget.h

```diff
@@ -130,6 +131,8 @@ class TerminalWidget final : public GUI::Frame
 
     bool m_should_beep { false };
     bool m_belling { false };
+    bool m_alt_key_held { false };
+    bool m_square_selection { false };
```

Comment:
`square` seems like a misnomer. It's really a rectangle, not necessarily a square. :)

Other than that, this seems fine! Does this also work if you are scrolled back in history?

Reply:
Yeah the selection seems to work in history, but copying text in scroll history with either a normal or ALT selection seems to be broken currently.